pyproject.toml

```diff
@@ -43,7 +43,7 @@ features = [
 ]
 
 [tool.hatch.envs.tests.scripts]
-test = "pytest -n auto {args}"
+test = "pytest -n auto {args} && hatch -e docs run docs:build"
```

Comment:
I think it's best if we've got a command for building docs instead of building them for every test run.

Reply:
yeah however the docs building also makes sure that the scripts in the docs run, so its a reliable test which ensures that everything works. its also quick. i dont wanna bloat the repo with too many specific commands tbh